backend/database/migrations/2019_08_07_114836_tracking_info_table.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class TrackingInfoTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('tracking_info', function (Blueprint $table) {
```

Comment:
I cannot understand why did you create table only with one id? This table is useless because one web site has one tracking number. And number should be saved in seprate field. Add to the website field "tracking_number" and fill it incrementally programmatically

Reply:
Fix